app/bundles/InstallBundle/Controller/InstallController.php

```diff
@@ -14,6 +14,7 @@
 use Doctrine\DBAL\DBALException;
 use Mautic\CoreBundle\Configurator\Configurator;
 use Mautic\CoreBundle\Controller\CommonController;
+use Mautic\InstallBundle\Helper\SchemaHelper;
```

Comment:
This was removed in https://github.com/mautic/mautic/pull/9739 so we should not be adding it back in (as far as I know at least!)

Reply:
This helper is not needed as well